### DISCUSSION TITLE
Is it possible to use kustomize to lookup into deployed K8s resource?

### DISCUSSION THREAD

**Discussion by jaythamke** [kubernetes-sigs/kustomize#5532] [Q&A]
Hi,
I am trying out `replacements` (former `vars`) feature (kustomize version v5.4.*) to replace the configuration from source to target. 
For me the source is a Kubernetes secret but already deployed in a cluster and I want to lookup the deployed secret data e.g. `.spec.data.foo` into the given target field.
I could successfully replace the placeholders if it is placed in a file. But I did not find how to fetch that secret from a deployed secret object. Is it really possible and how it can be achieved? Thanks

**Comment by ChristianCiach** [comment]
It's not possible. Kustomize is a manifest generator, not a deployment tool. Kustomize never connects to a Kubernetes cluster.
